dotlottie/src/main/java/com/lottiefiles/dotlottie/core/compose/ui/DotLottieAnimation.kt

```diff
@@ -144,19 +147,34 @@ fun DotLottieAnimation(
         }
     }
 
-    LaunchedEffect(loop, autoplay, playMode, useFrameInterpolation, speed, segments) {
+    LaunchedEffect(
+        loop,
+        autoplay,
+        playMode,
+        useFrameInterpolation,
+        speed,
+        segments,
+        theme,
+        marker
+    ) {
         val conf = dlPlayer.config()
         conf.loopAnimation = loop
         conf.autoplay = autoplay
         conf.mode = playMode
         conf.useFrameInterpolation = useFrameInterpolation
         conf.speed = speed
+        conf.marker = marker ?: ""
         if (segments != null) {
             conf.segments = listOf(segments.first, segments.second)
         } else {
             conf.segments = emptyList()
         }
 
+        if (theme != null) {
+            dlPlayer.loadTheme(theme)
+        } else {
+            // @TODO: Unload theme. Feature missing in Thorvg
```

Comment:
to minimize the no of functions we have in dotlottie core player, maybe we can reuse the same loadTheme to unload as well if we provide an empty theme_id 
```suggestion
            // @TODO: Unload theme. Feature missing in Thorvg
            dlPlayer.loadTheme("")
```